sentry-samples/sentry-samples-spring-boot-jakarta/src/main/java/io/sentry/samples/spring/boot/jakarta/SentryDemoApplication.java

```diff
@@ -41,31 +36,36 @@ RestClient restClient(RestClient.Builder builder) {
   }
 
   @Bean
-  public JobDetailFactoryBean jobDetail() {
-    JobDetailFactoryBean jobDetailFactory = new JobDetailFactoryBean();
-    jobDetailFactory.setJobClass(SampleJob.class);
-    jobDetailFactory.setDurability(true);
-    jobDetailFactory.setJobDataAsMap(
-        Collections.singletonMap(SENTRY_SLUG_KEY, "monitor_slug_job_detail"));
-    return jobDetailFactory;
-  }
-
-  @Bean
-  public SimpleTriggerFactoryBean trigger(JobDetail job) {
-    SimpleTriggerFactoryBean trigger = new SimpleTriggerFactoryBean();
-    trigger.setJobDetail(job);
-    trigger.setRepeatInterval(2 * 60 * 1000); // every two minutes
-    trigger.setRepeatCount(SimpleTrigger.REPEAT_INDEFINITELY);
-    trigger.setJobDataAsMap(
-        Collections.singletonMap(SENTRY_SLUG_KEY, "monitor_slug_simple_trigger"));
-    return trigger;
+  public CacheManager cacheManager() {
+    return new SentryCacheManagerWrapper(new ConcurrentMapCacheManager("people"));
```

Comment:
Could also do this in a bean post processor to support the default bean. We'll consider the bean post processor approach but might end up integrating this into our POTel offering.